src/accelerate/utils/transformer_engine.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2022 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch.nn as nn
+
+from .imports import is_fp8_available
+
+
+if is_fp8_available():
+    import transformer_engine.pytorch as te
+
+
+def convert_model(model, to_transformer_engine=True, _convert_linear=True, _convert_ln=True):
```

Comment:
Should this run an explicit try/catch for `is_fp8_available` and raise an error if not?

Reply:
It shouldn't be called if `is_fp8_available` is `False`.